### PR TITLE
ADT_Assignment_25_Done_8Mins

### PR DESCRIPTION
**Pull request by AditiGJUN20**
### Assignment 25:
Time: 8 Mins
image

### CODE REVIEW

**Comment by AmitaRJun20** [comment]
A Clean code...Good job...

**Review by AmitaRJun20 — APPROVED** [review]
A clean code...Good job

File: src/aditiGJUN20/DigitSumInString.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ */
+
+package aditiGJUN20;
+
+public class DigitSumInString {
+	int getSumOfAllDigits(String string) {
+		int sum=0;
+		for(int index=0;index<string.length();index++) {
+ 			if(Character.isDigit(string.charAt(index)))
+ 				sum=sum+Integer.parseInt(String.valueOf(string.charAt(index)));
```
Comment (KrishnaKTechnocredits) [inline src/aditiGJUN20/DigitSumInString.java:15]: We can use Character class method here. 
Character.getNumericValue(ch)

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

File: src/aditiGJUN20/DigitSumInString.java (hunk 1 above)
Comment (AditiGJUN20) [reply src/aditiGJUN20/DigitSumInString.java:15]: Thank you @KrishnaKTechnocredits 
I was not aware about the method which you've mentioned. 
I'll surely try to accommodate it my upcoming assignments.

**Pull request merged**